contracts/argent_account.cairo

```diff
@@ -24,9 +24,9 @@ mod ArgentAccount {
     #[external]
     fn initialize(signer: felt, guardian: felt, guardian_backup: felt) {
         // check that we are not already initialized
-        assert(signer::read() == 0, 'argent: already initialized');
+        assert(signer::read() == 0, 'argent/already-initialized');
         // check that the target signer is not zero
-        assert(signer != 0, 'argent: signer cannot be null');
+        assert(signer != 0, 'argent/null-signer');
```

Comment:
question for @gaetbout here: it seems the name `signer` refers to either the storage variable or the function argument depending on how it's used. is this how it works in rust too?

this seems a bit dangerous to me, should we prefix all our function arguments with `_` like we do in solidity?

Reply:
I'd rather keep storage as is and rename method args

Reply:
We can simply use `new_signer` for the argument if that makes it easier.

Reply:
I'll prefix the arguments with `initial_`

Reply:
I thin new_XXX makes more sense, initial_XXX makes me think that it is the initial value that needs to be changed

Reply:
fine then. there's also an unused `guardian_backup` arg that I'll remove